editoast/src/views/paced_train.rs

```diff
@@ -229,20 +234,36 @@ async fn simulation_summary(
 )]
 async fn get_path(
     State(AppState {
-        db_pool: _db_pool,
-        valkey: _valkey_client,
-        core_client: _core,
+        db_pool,
+        valkey: valkey_client,
+        core_client: core,
```

Comment:
Can you do something similar than what is mentioned [here](https://github.com/OpenRailAssociation/osrd/pull/11022#discussion_r1983442071)?